### PR TITLE
feat: Add use cases

### PR DESCRIPTION
**Pull request by bharathkkb**
Example uses cases for SIEM

### CODE REVIEW

**Comment by bharathkkb** [comment]
@daniel-cit these are the example uses cases Aaron wrote.

File: modules/bq-log-alerting/use-cases/iam_role_add.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SELECT
+  receiveTimestamp,
+  timestamp AS eventTimestamp,
+  protopayload_auditlog.requestMetadata.callerIp,
+  protopayload_auditlog.authenticationInfo.principalEmail,
+  protopayload_auditlog.resourceName
```
Comment (atos-cit) [inline modules/bq-log-alerting/use-cases/iam_role_add.sql:20]: Missing a comma: 

`protopayload_auditlog.resourceName`

File: modules/bq-log-alerting/use-cases/non_allowlisted_services.sql [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  SELECT
+    receiveTimestamp,
+    timestamp,
+    protopayload_auditlog.requestMetadata.callerIp AS callerIp,
+    protopayload_auditlog.authenticationInfo.principalEmail AS principalEmail,
+    logName,
+    log.resource.labels.project_id AS projectId,
+    REGEXP_EXTRACT(authInfo.resource, r '^services/([^/]*)') AS serviceName
+  FROM
+    `${project}.raw_logs.cloudaudit_googleapis_com_activity_*` log,
```
Comment (atos-cit) [inline modules/bq-log-alerting/use-cases/non_allowlisted_services.sql:25]: Missing `AS` in

`    `${project}.raw_logs.cloudaudit_googleapis_com_activity_*` AS log,`

**Comment by atos-cit** [comment]
@bharathkkb Yes I can. I commiting every use case in #58 so after I finish I can decline this one.

**Comment by morgante** [comment]
@bharathkkb @atos-cit Do we still need this?

**Comment by bharathkkb** [comment]
superseded by #77

**Pull request closed without merging**